onnxscript/function_libs/torch_aten/ops/core.py

```diff
@@ -4353,9 +4355,11 @@ def aten_sub(self, other, alpha: float = 1) -> TensorType:
     # sub.Tensor(Tensor self, Tensor other, *, Scalar alpha=1) -> Tensor
 
     if alpha != 1:
-        other = op.Mul(other, alpha)  # type: ignore[arg-type]
+        other_scaled = op.Mul(other, alpha)  # type: ignore[arg-type]
+    else:
+        other_scaled = other
```

Comment:
Same comment as above.

Reply:
Resolved to focus on the discussion above.